clang/test/CIR/Lowering/ThroughMLIR/scope.cir

```diff
@@ -21,26 +20,6 @@ module {
 // MLIR-NEXT:   }
 // MLIR-NEXT:   return
 
-
-//       LLVM: define void @foo()
-//  LLVM-NEXT:   %1 = call ptr @llvm.stacksave()
-//  LLVM-NEXT:   br label %2
-// LLVM-EMPTY:
-//  LLVM-NEXT: 2:
-//  LLVM-NEXT:   %3 = alloca i32, i64 1, align 4
-//  LLVM-NEXT:   %4 = insertvalue { ptr, ptr, i64 } undef, ptr %3, 0
-//  LLVM-NEXT:   %5 = insertvalue { ptr, ptr, i64 } %4, ptr %3, 1
-//  LLVM-NEXT:   %6 = insertvalue { ptr, ptr, i64 } %5, i64 0, 2
-//  LLVM-NEXT:   %7 = extractvalue { ptr, ptr, i64 } %6, 1
-//  LLVM-NEXT:   store i32 4, ptr %7, align 4
-//  LLVM-NEXT:   call void @llvm.stackrestore(ptr %1)
```

Comment:
I was thinking about attempting to make this work again just to not degrade the ThroughMLIR pipeline any more than it currently is. Did you look at the actual problem?

Reply:
@lanza as far as I tested, this test was actually passing. When running LIT it was listed as "Unexpectedly Passed", meaning the `XFAIL` shouldn't be necessary. Although, it might be working only on my environment. Is this test still crashing for you?

I also opted to remove the `mlir-translate` test since this is not a direct part of ClangIR. Though I just realized that I should probably have left in the `RUN: cir-opt %s -cir-to-mlir -cir-mlir-to-llvm -o -` which is a part of ClangIR. I'll add it back.

Reply:
> @lanza as far as I tested, this test was actually passing. When running LIT it was listed as "Unexpectedly Passed", meaning the XFAIL shouldn't be necessary. Although, it might be working only on my environment. Is this test still failing for you?

Ah okay, I added an extra xfail in throughmlir but I didn't pay attention to which. 

> I also opted to remove the mlir-translate test since this is not a direct part of ClangIR. Though I just realized that I should probably have left in the RUN: cir-opt %s -cir-to-mlir -cir-mlir-to-llvm -o - which is a part of ClangIR. I'll add it back.

Good point!

Reply:
> I should probably have left in the `RUN: cir-opt %s -cir-to-mlir -cir-mlir-to-llvm -o -`

Fixed!

Reply:
On second thought, I'm not sure if this test is useful. despite `-cir-mlir-to-llvm` being a ClangIR-specific pass, it only uses rewrite patterns from other dialects. The only thing we are actually testing here from our code is the `ConvertMLIRToLLVMPass` configuration.